src/components/App/App.tsx

```diff
@@ -34,10 +34,13 @@ const App: React.FC = () => {
       <header>Restaurant Finder</header>
       <main> 
         <SearchAndFilter 
-          data={restaurants} 
+          allData={restaurants} 
           columns={['name', 'city', 'state', 'telephone', 'genre', 'website']}
           filterTypes={['state', 'genre']}
         />
+        {isLoading &&
+          <img src="/loading.gif" alt="loading icon" />
```

Comment:
I really love this icon, but I don't think there's enough data yet to ever see it 😟